src/components/body/components/sideBar/components/teamMateList/TeamMateList.tsx

```diff
@@ -0,0 +1,66 @@
+//libs
+import { useCallback, useState } from "react";
+
+//component
+import { TeamMateDetail } from "./components/teamMateDetail";
+import { Error } from "../../../../../error";
+import { Spinner } from "../../../../../spinner";
+
+//hooks
+import { useTeamMatesQuery } from "../../../../../../hooks/useTeamMatesQuery";
+import { useUserId } from "../../../../../../hooks/UserContext";
+
+//type
+import { User } from "../../../../../../types/types";
+
+//style
+import styles from "./TeamMateList.module.css";
+
+type TeamMateListProps = {
+  onChangeActiveTeamMate: (activeTeamMate: User) => void;
+  activeTeamMate: User;
+};
+
+export const TeamMateList = ({
+  onChangeActiveTeamMate,
+  activeTeamMate,
+}: TeamMateListProps) => {
+  const [toggleTeamMateList, setToggleTeamMateList] = useState<boolean>(true);
```

Comment:
Names should depict exactly what data/state is stored in them.
`toggleTeamMateList` -> From the name it looks like it's a handler that toggles teamMateList.

keep the name as `isTeamMateListCollapsed`.